Library/Homebrew/extend/os/linux/dev-cmd/bottle.rb

```diff
@@ -0,0 +1,51 @@
+# typed: false
+# frozen_string_literal: true
+
+module Homebrew
+  extend T::Sig
+
+  module_function
+
+  def setup_tar_and_args!(args, mtime)
+    # Without --only-json-tab bottles are never reproducible
+    default_tar_args = ["tar", [].freeze].freeze
+    return default_tar_args unless args.only_json_tab?
+
+    # Ensure tar is set up for reproducibility.
+    # https://reproducible-builds.org/docs/archives/
+    gnutar_args = [
+      "--format", "pax", "--owner", "0", "--group", "0", "--sort", "name", "--mtime=#{mtime}",
+      # Set exthdr names to exclude PID (for GNU tar <1.33). Also don't store atime and ctime.
+      "--pax-option", "globexthdr.name=/GlobalHead.%n,exthdr.name=%d/PaxHeaders/%f,delete=atime,delete=ctime"
+    ].freeze
+
+    ["tar", gnutar_args].freeze
+  end
+
+  def formula_ignores(f)
```

Comment:
Nice work! I think both these methods would ideally call the existing ones first e.g. with the `alias generic_setup_tar_and_args! setup_tar_and_args!` patterns you can see used elsewhere. Shout if you want help with that!